### PR TITLE
SE-2: Fix form item width on cases activity panel

### PR DESCRIPTION
**Pull request by swastikpareek**
## Overview
This PR fixes the form item widths and make them standard to 230px wide on civicases activity edit panel.

## Before
Screenshot 2020-01-02 at 4 22 35 PM
Screenshot 2020-01-02 at 4 23 01 PM
Screenshot 2020-01-02 at 4 23 14 PM

## After
Screenshot 2020-01-02 at 4 14 20 PM
Screenshot 2020-01-02 at 4 14 39 PM
Screenshot 2020-01-02 at 4 14 51 PM

## Technical Details
The standard width of the cases activity panel form item (except description) is made 230px wide. 
* `.crm-form-date` uses date picker library to provide width to the input element and due to some reason the calculation of width by the library is done on the bases of (width -42) to the input element ( remaining 34px is being taken by the icon) So we had to use this special equation to equate the width `civicase-activity-panel-form-date-item-width + (42px - 34px)`

* For the follow up activity subject form item there are no helper class added to it or it's parent (from core civicrm only) so we had to specifically target this item using the attribute selector `input[name='followup_activity_subject']`

## Update
In [this commit](https://github.com/compucorp/uk.co.compucorp.civicase/pull/319/commits/7675706fb46859f1737aecee8993d08ae22db125) changed the width of the time field to accommodate `am/pm` suffix when date is changed to 12 hour format
#### Before
Screenshot 2020-01-03 at 6 10 13 PM
#### After
Screenshot 2020-01-03 at 6 08 56 PM

### CODE REVIEW

File: scss/components/_activity-panel.scss [hunk 1]
```diff
@@ -194,5 +197,43 @@
...
+  // Some how the final width applied is 168 - 42 to the input element
+  // and 34px to date picker icon = which counts to: 160px.
+  .crm-form-date {
+    width: $civicase-activity-panel-form-date-item-width + (42px - 34px);
+  }
+
+  // Remaining space (230 - 160px) - 70 - 10px of margin.
+  .crm-form-time {
+    margin-left: 10px;
+    width: $civicase-activity-panel-form-item-width - $civicase-activity-panel-form-date-item-width - 10px;
```
Comment (deb1990) [inline scss/components/_activity-panel.scss]: The comment says `(230 - 160px) - 70 - 10px`. But looks like in the calculation 70px is missing, is the comment correct?

Comment (swastikpareek) [reply scss/components/_activity-panel.scss]: Oops. It should be `=70` rather.

**Comment by jamienovick** [comment]
@swastikpareek the images look good to me, the only thing is that I can see in one image that the 2019 in the date field is duplicated. I don't think this is relevant to your changes but can you just double check?

**Comment by swastikpareek** [comment]
> @swastikpareek the images look good to me, the only thing is that I can see in one image that the 2019 in the date field is duplicated. I don't think this is relevant to your changes but can you just double check?
> 
> Actually sorry just seen - the "time" field needs to have space for the following characters:
> 
> HH:mm am - i.e. you can sometimes have am/pm after the time. Will your approach work for date time in that format? You can adjust the date / time formats at administer-> localisation -> date formats and select time input format to be 12 hours.
> 
> Could you check it works with that also? Thanks

@jamienovick : Fixed the width of date field. Now it looks better. Please see the updated section of PR details.

**Comment by jamienovick** [comment]
@swastikpareek thanks noted and approved.

**Pull request merged**